checkpoints/check_audio.py

```diff
@@ -59,7 +59,8 @@ def get_sound_cards():
 
 def is_service_active():
     """Returns True if the voice-recognizer service is active."""
-    output = subprocess.check_output(['systemctl', 'show', SERVICE_NAME]).decode('utf-8')
+    output = subprocess.check_output(['systemctl', 'show',
```

Comment:
We've been working to 100 character lines, so the presubmit shouldn't complain if you set your pyformat to 100 characters. Can you un-wrap the wrapped lines in this PR?

If you figure out a good pyformat config, let's add it to HACKING.md. So far I've been formatting manually.